docs-src/app-dev-context/namespaces.md

```diff
@@ -0,0 +1,18 @@
+---
+id: namespaces
+title: How to create Namespaces
+sidebar_label: Namespaces
+description: A Namespace is a unit of isolation within the Temporal Platform.
+tags:
+  - guide-context
+---
+
+A Namespace is a unit of isolation within the Temporal Platform.
+
+You can use Namespaces to match the development lifecycle; for example, having separate `dev` and `prod` Namespaces.
+Or you could use them to ensure Workflow Executions between different teams never communicate; such as ensuring that the `teamA` Namespace never impacts the `teamB` Namespace.
+
+The register and update Namespace APIs send a gRPC request to the Frontend Service to create or update a Namespace.
+Use a custom authorizor on your Frontend Service in the Temporal Cluster to set restrictions on who can create or update Namespaces.
```

Comment:
I think this is the same Authorizer Plugin we have defined in terms?
If so - let us link to it.

Reply:
Yes! linked